.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: Publish
+
+on:
+  push:
+
+jobs:
+
+  publish:
+    if: startsWith(github.ref, 'refs/tags') || github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # chartpress requires the full history
+          fetch-depth: 0
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Install dependencies and run chartpress
+        run: pip install --no-cache-dir -r dev-requirements.txt
+      - name: Publish chart
+        run: |
+          . ./ci/common
+          ./ci/publish
+        env:
+          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
There are environment variables used to decrypt an encrypted SSH key registered with the jupyterhub/helm-chart repo so we can push to it. We have to either replace that SSH key with a new one we have encrypted or create a new deploy key for jupyterhub/helm-chart, encrypt it, and use an environment variable as key to decrypt it.

Let's create a new SSH public/private key, encrypt the private in this repo using some secret, and register the public part for push access in the jupyterhub/helm-chart repo here: https://github.com/jupyterhub/helm-chart/settings/keys.

Or, perhaps there is a GitHub action way to help us get a SSH key encrypted into the repo? Ah yes! Let's try out this action perhaps? https://github.com/marketplace/actions/install-ssh-key

Reply:
I added a commit and registered a deployment key etc, but credentials for dockerhub remain still. I have asked @minrk for help about this.

> @minrk I'd like to provide GitHub actions the password for the dockerhub jupyterhubbot account in order to let pushes for z2jh work. This will probably also be done for binderhub in the end, could you provide me with the password or add DOCKER_USERNAME and DOCKER_PASSWORD here:
>
>https://github.com/jupyterhub/zero-to-jupyterhub-k8s/settings/secrets/actions
>https://github.com/jupyterhub/binderhub/settings/secrets/actions